### DISCUSSION TITLE
Admin changelogs

### DISCUSSION THREAD

**Discussion by PJB3005** [space-wizards/space-station-14#24054] [Code announcements]
The changelog system should now support adding to the **admin changelog**. This is a separate changelog that's only visible to admins. (This is intended to avoid clutter for players, not to "hide" things from players alright.)

To write entries for the admin changelog, you use the existing system but put "ADMIN:" above them. Like so:

```
:cl:
- add: Added fun

ADMIN:
- remove: Actually, the fun was removed
```

\* assuming I set everything up right at least.

**Comment by Just-a-Unity-Dev** [comment]
Would be cool if this was still available to players with a checkbox or something

**Reply by deltanedas** [reply]
yeah prs arent exactly for your eyes only top secret
